software/kallisto/index/main.nf

```diff
@@ -0,0 +1,39 @@
+// Import generic module functions
+include { initOptions; saveFiles; getSoftwareName } from './functions'
+
+params.options = [:]
+options        = initOptions(params.options)
+
+process KALLISTO_INDEX {
+    tag "$fasta"
+    label 'process_medium'
+    publishDir "${params.outdir}",
+        mode: params.publish_dir_mode,
+        saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), publish_id:'') }
+
+    conda (params.enable_conda ? "bioconda::kallisto=0.46.2" : null)
+    if (workflow.containerEngine == 'singularity' && !params.singularity_pull_docker_container) {
+        container "https://depot.galaxyproject.org/singularity/kallisto:0.46.2--h4f7b962_1"
+    } else {
+        container "quay.io/biocontainers/kallisto:0.46.2--h4f7b962_1"
+    }
+
+    input:
+    path(fasta)
+
+    output:
+    tuple path("*.idx"), emit: idx
+    path "*.version.txt"          , emit: version
```

Comment:
```suggestion
    tuple path("*.idx") , emit: idx
    path "*.version.txt", emit: version
```

Reply:
Be nice if we can output a directory instead like with Salmon and all of the other aligners: https://github.com/nf-core/modules/blob/acb1a12a56f6eca1d9a30b86ca880d39af0ba5d6/software/salmon/index/main.nf#L26

Reply:
Ok sure, I'll do it like that then